api/views/recordings_views.py

```diff
@@ -4,7 +4,7 @@
 from drf_spectacular.utils import extend_schema
 from rest_framework import status
 
-from api.serializers import InstructorRecordingsSerializer, RecordingTitleUpdateSerializer
+from api.serializers import *
```

Comment:
Let's revert this back; it's always better to list the classes/functions that are imported.